x-pack/plugins/actions/server/builtin_action_types/case/types.ts

```diff
@@ -4,51 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { TypeOf } from '@kbn/config-schema';
-import {
-  IncidentConfigurationSchema,
-  MapRecordSchema,
-  CommentSchema,
-  EntityInformationSchema,
-} from './schema';
-
-export type IncidentConfiguration = TypeOf<typeof IncidentConfigurationSchema>;
-export type MapRecord = TypeOf<typeof MapRecordSchema>;
-export type Comment = TypeOf<typeof CommentSchema>;
-export type EntityInformation = TypeOf<typeof EntityInformationSchema>;
-
+export interface SimpleComment {
```

Comment:
Can we move the types outside of the case folder so we can delete it? Even if that means that we would duplicate the type on each connector.